src/main/java/frc/robot/commands/assist/PrepareForScore.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.commands.assist;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.arm.Arm;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class PrepareForScore extends SequentialCommandGroup {
+  // TODO:  read arm angle and shooter velocity from GUI
```

Comment:
Similarly, add a comment describing the expected behavior:

Prepare for Score

1. Moves arm to ideal angle for shooting
2. Turns on shooter to ideal speed for scoring